File: app/src/main/java/com/ohmcoe/vultr/MyToast.java
package com.ohmcoe.vultr;


import android.content.Context;
import android.widget.Toast;

class MyToast {
    private Context toastContext;
    private CharSequence text;

    MyToast(Context context, String str) {
        toastContext = context;
        text = str;
    }

    public void setText(String str)
    {
        text = str;
    }

    public void show() {
        Toast toast = Toast.makeText(toastContext, text, Toast.LENGTH_LONG);
        toast.show();
    }
}

File: app/src/main/java/com/ohmcoe/vultr/MyToast.kt
package com.ohmcoe.vultr


import android.content.Context
import android.widget.Toast

internal class MyToast(private val toastContext: Context, str: String) {

    private var text: CharSequence? = null

    init {
        text = str
    }

    fun setText(str: String) {
        text = str
    }

    fun show() {
        val toast = Toast.makeText(toastContext, text, Toast.LENGTH_LONG)
        toast.show()
    }
}
